web/templates/recipient_row.hbs

```diff
@@ -70,8 +70,8 @@
                         <i class="zulip-icon zulip-icon-mute-new recipient_bar_icon" data-tippy-content="{{t 'You have muted this topic.'}}"
                           role="button" tabindex="0" aria-haspopup="true" aria-label="{{t 'You have muted this topic' }}"></i>
                     {{else}}
-                        <i class="zulip-icon zulip-icon-inherit recipient_bar_icon" data-tippy-content="{{t 'You will get default notifications for this topic.'}}"
-                          role="button" tabindex="0" aria-haspopup="true" aria-label="{{t 'You will get default notifications for this topic' }}"></i>
+                        <i class="zulip-icon zulip-icon-inherit recipient_bar_icon" data-tippy-content="{{t 'Notifications are based on your configuration for'}}"
+                          role="button" tabindex="0" aria-haspopup="true" aria-label="{{t 'Notifications are based on your configuration for' }} {{stream_name}} channel"></i>
```

Comment:
You can use the variable directly inside the translation helper like such:
`"{{t 'Notifications are based on your configuration for {stream_name} channel' }}"`

Reply:
Not breaking translated phrases up too much is really important. For example, in other languages, "channel" might take different forms, depending on where it is in the sentence.